Configuration/DataProcessing/python/Reco.py

```diff
@@ -157,8 +157,12 @@ def visualizationProcessing(self, globalTag, **args):
 
         eiStep=''
 
-        options.step += 'RAW2DIGI,L1Reco,RECO'+eiStep+',ENDJOB'
-
+        if 'beamSplashRun' in args:
+          recoDQMoptions = ":localreco+hcalOnlyGlobalRecoSequence+caloTowersRec" if args['beamSplashRun'] else ""
```

Comment:
Shouldn't this just replace the `if ... else:` structure?

Reply:
hi, not clear, the first check is if 'beamSplashRun' in args, the second one to check if args['beamSplashRun'] is True,
so, it could be:
```
if 'beamSplashRun' in args:
  if args['beamSplashRun']  :
     recoDQMoptions = ":localreco+hcalOnlyGlobalRecoSequence+caloTowersRec"
  else : 
     recoDQMoptions = ""
```

Reply:
You already updated it. Anyhow, just for my understanding: isn't adding `--beamSplashRun` as argument the same as checking "if args['beamSplashRun'] is True"?

Reply:
Following code will return False and then True:
```
def a(**args):
   print( args["a"] == True )
a(a=False)
a(a=True)
```
Hope this helps.

Reply:
cmsRun options are not propagated to the visualizationProcessing, instead the function is called at line:
https://github.com/cms-sw/cmssw/blob/1fdb41b21c9a67094be0218604057c80e01abba1/DQM/Integration/python/clients/visualization-live-secondInstance_cfg.py#L64